Header header
float64 latitude      #degree
float64 longitude     #degree
float32 altitude      #WGS 84 reference ellipsoid fused by barometer
uint16  visibleSatelliteNumber #number of visible satellites